webapp-django/crashstats/crashstats/views.py

```diff
@@ -1050,14 +1049,23 @@ def crashes_per_user(request, default_context=None):
     default = request.session.get('date_range_type', 'report')
     context['date_range_type'] = params.get('date_range_type') or default
 
-    api = SuperSearchUnredacted()
-    # XXX HACK! Because of a bug in SuperSearch we can't search for
-    # '4X.0b' so we have to change that to '4X.0b1' :(
-    # See https://bugzilla.mozilla.org/show_bug.cgi?id=1197251
+    params['original_versions'] = params['versions'][:]  # make a copy
```

Comment:
Well, I just find that ``[:]`` is a lot more cryptic than ``list()``. If you're not well versed into Python, it's difficult to understand the purpose of ``[:]``. With ``list()`` it's clear that you are creating a new object. I understand both, but I tend to use ``list()`` because I think it is simpler to read and understand. 

But I do not care enough to argue more than just that, so it's your call, really. :) (And I do not really mind having both ways in the code base, it's just a styling detail. )